components/PrimarySourceSetsComponents/SingleSet/SourceSetInfo/index.js

```diff
@@ -49,34 +49,22 @@ class SourceSetInfo extends React.Component {
                   }}
                 />
                 <div className={classNames.bannerTextWrapper}>
-                  <h1
-                    dangerouslySetInnerHTML={{
-                      __html: markdownit.renderInline(set.name)
-                    }}
-                    className={classNames.bannerTitle}
-                  />
+                  <h1 className={classNames.bannerTitle}>
+                    <ReactMarkdown
+                      source={set.name}
+                      allowedTypes={["emphasis"]}
+                      unwrapDisallowed
+                    />
+                  </h1>
                 </div>
               </div>
-              <div
+              <ReactMarkdown
                 id="dpla-description"
+                source={set.hasPart.find(item => item.name === "Overview").text}
                 className={`${classNames.description} sourceSetDescription ${classNames.description} ${this
                   .state.isOpen
                   ? classNames.open
                   : ""}`}
-                dangerouslySetInnerHTML={{
-                  __html: markdownit.render(
-                    set.hasPart
-                      .find(item => item.name === "Overview")
-                      .text.replace(
-                        /https?:\/\/.*?\/primary-source-sets\/sources\//g,
-                        "sources/"
-                      )
-                      .replace(
-                        /https?:\/\/.*?\/primary-source-sets\/sets\//g,
-                        "/primary-source-sets/"
-                      )
-                  )
-                }}
               />
```

Comment:
Did you want to allow emphasis in the above `ReactMarkdown`?

Reply:
yes and it is because all types are allowed by default. emphasis is specified in the cases where we only want emphasis to show up